src/compile/scale/range.ts

```diff
@@ -219,8 +239,15 @@ export function defaultRange(
       if (scaleType === 'ordinal') {
         // Only nominal data uses ordinal scale by default
         return type === 'nominal' ? 'category' : 'ordinal';
+      } else if (isContinuousToDiscrete(scaleType)) {
+        if (config.range && isVgScheme(config.range.ordinal)) {
+          return config.range.ordinal;
```

Comment:
I get what you try to do now, but I think you should return

```
{
   ...config.range.ordinal,
   count
}
```

instead :)

Reply:
Yup!